src/index.js

```diff
@@ -1,2 +1,172 @@
+/*eslint-disable*/
 import _ from 'lodash';
 import './style.css';
+import myObject from './modules/function';
+
+const whole = document.querySelector('.toDo');
+const textInput = document.querySelector('.addtext');
+const form = document.querySelector('.form1');
+const submit = document.querySelector('.submit');
+const todoCon = document.querySelector('.todo-container');
+
+// class object
+
+const Array1 = [];
+const addToList = (todovalue) => {
+  const todocontainer = document.createElement('div');
+  todocontainer.className = 'todocontainer';
+  todocontainer.innerHTML += `
+    <input type="checkbox" class="check">
+    <span>${todovalue}</span>
+    <i class="fa-solid fa-ellipsis-vertical menu"></i>
+    <i class="fa-solid fa-trash-can delete"></i>
+  `;
+  todoCon.appendChild(todocontainer);
+
+  const checkbox = document.querySelectorAll('.check');
+  checkbox.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.toggle('checkedcontainer');
+      i.nextElementSibling.classList.toggle('inputvalue');
+      i.parentElement.lastElementChild.classList.toggle('trash');
+      i.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-disable');
+      updateLocal();
+    });
+  });
+
+  const object = new myObject(todovalue, false, checkbox.length - 1);
+  Array1.push(object);
+  localStorage.setItem('activities', JSON.stringify(Array1));
```

Comment:
You can create external functions for setting and getting local storage to avoid repeating ` localStorage.setItem('activities', JSON.stringify(Array1))`